### PR TITLE
feat: add es.json

### LINKED ISSUE #4060: Add Spanish 🇪🇸
Now that Spanish is nearly finished thanks to @FrankyFFV, let's add it to Suite. 

Please download the Spanish translation from crowdin and add it to the language switcher in Suite. Please add a beta tag so it says `Spanish (BETA)`.

### PR DESCRIPTION
**Pull request by mroz22**
resolve #4060

### CODE REVIEW

File: packages/suite-data/files/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,1489 @@
...
+	"TR_YOUR_NEW_VERSION": "La versión {version} está disponible.",
+	"TR_YOUR_TREZOR_IS_ALMOST_READY": "Tu Trezor está casi listo",
+	"TR_YOUR_TREZOR_IS_NOT_BACKED_UP": "Tu Trezor no está respaldado.",
+	"TR_YOUR_WALLET_IS_READY_WHAT": "Tu Wallet está lista. ¿Qué hacer ahora?",
+	"TR_YOU_CAN_SELECT_EITHER": "Puedes seleccionar un nuevo Trezor o cualquier otro dispositivo Trezor que haya sido utilizado e inicializado antes.",
+	"TR_YOU_EITHER_HAVE_T1": "Tienes una semilla que contiene 12, 18 o 24 palabras.",
+	"TR_YOU_EITHER_HAVE_T2": "Tienes una semilla que contiene 12, 18, 20, 24 o 33 palabras.",
+	"TR_YOU_MAY_CHOSE_EITHER_STANDARD": "You might chose either standard backup type or Shamir backup",
+	"TR_YOU_WERE_DISCONNECTED_DOT": "Has sido desconectado.",
+	"TX_CONFIRMATIONS_EXPLAIN": "TODO TODO TODO ExPlAnAtIoN"
```
Comment (mroz22) [inline packages/suite-data/files/translations/es.json]: no es finito hombres karamba

Comment (mroz22) [reply packages/suite-data/files/translations/es.json]: @FrankyFFV :smile:

Comment (mroz22) [reply packages/suite-data/files/translations/es.json]: otherwise excellent work :+1:

Comment (FrankyFFV) [reply packages/suite-data/files/translations/es.json:1488]: I don’t understand your comment 😂

File: packages/suite-data/files/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,1489 @@
...
+	"TR_HIDDEN_WALLET_DESCRIPTION": "Requiere frase de contraseña",
+	"TR_HIDDEN_WALLET_MODAL_DESCRIPTION": "Extensión de la wallet derivada de la semilla de recuperación por tu propia entrada elegida.",
+	"TR_HIDDEN_WALLET_TOOLTIP": "Extension of the wallet derived from recovery seed by your own chosen input.",
+	"TR_HIDE_ADVANCED_OPTIONSHide advanced sending form": "Ocultar opciones avanzadas",
+	"TR_HIDE_BUTTON": "Ocultar",
+	"TR_HIDE_GRAPH": "Ocultar gráfico",
+	"TR_HOLD_BOTH_BUTTONS": "Hold both buttons while connecting device",
+	"TR_HOLD_LEFT_BUTTON": "Mantén pulsado el botón de la izquierda o ambos botones mientras conectas el dispositivo",
+	"TR_HOLOGRAM_STEP_ACTION_NOT_OKButton to click when hologram looks different": "Mi holograma parece diferente",
+	"TR_HOLOGRAM_STEP_ACTION_OKButton to click in alright case": "Mi holograma está bien",
```
Comment (marekrjpolak) [inline packages/suite-data/files/translations/es.json]: These translation keys doesn't look good. I guess that the transforming script could be broken and handled translations with description incorrectly.

**Review by marekrjpolak — CHANGES_REQUESTED** [review]

File: packages/suite-data/files/translations/es.json (hunk 2 above)
Comment (FrankyFFV) [reply packages/suite-data/files/translations/es.json]: It's true, are good in Crowdin but not here.

Comment (mroz22) [reply packages/suite-data/files/translations/es.json]: uf, thanks, good catch. It should be fixed now

Comment (marekrjpolak) [reply packages/suite-data/files/translations/es.json]: Looks good.

**Comment by marekrjpolak** [comment]
Is this a typo or an artistic intent? :)
image

File: packages/suite-data/files/translations/es.json (hunk 1 above)
Comment (marekrjpolak) [reply packages/suite-data/files/translations/es.json]: TX_CONFIRMATIONS_EXPLAIN is not used anywhere, so it's not a problem.

**Comment by FrankyFFV** [comment]
> Is this a typo or an artistic intent? :)
> image

Gramatically the correct expression to include both genders is "Bienvenido/a". However, the @ is also widely used (especially in IT).

**Comment by marekrjpolak** [comment]
> Gramatically the correct expression to include both genders is "Bienvenido/a". However, the @ is also widely used (especially in IT).

Oh, didn't know that, that's cool :)

**Review by marekrjpolak — APPROVED** [review]

**Review by marekrjpolak — APPROVED** [review]
LGTM.

**Pull request merged**